src/main/java/com/toamistaketracker/ToaMistake.java

```diff
@@ -168,9 +189,21 @@ private static String getStackingChatMessage(String message, int mistakeCount) {
         return sb.toString();
     }
 
-    private static String getAlternatingChatMessage(String message, int mistakeCount) {
-        String[] messageChoices = message.split("\\|");
+    private static String getAlternatingChatMessage(String message, String defaultMessage, int mistakeCount) {
+        String[] messageChoices = Arrays.stream(message.split("\\|")).filter(msg -> !msg.isEmpty())
+                .toArray(String[]::new);
+        if (messageChoices.length < 1) {
+            return defaultMessage;
+        }
         String messageChoice = messageChoices[mistakeCount % messageChoices.length];
-        return messageChoice;
+        // Truncate message length to prevent overly-spammy messages taking up too much screen space
+        return truncateMessage(messageChoice);
```

Comment:
See comment above. These helper methods don't need to worry about the truncation logic.